src/components/containers.js

```diff
@@ -47,3 +47,20 @@ export function Actions({ children, direction = 'row', classes = '' }) {
   const baseClass = `Hyp-Actions--${direction}`;
   return <div className={classnames(baseClass, classes)}>{children}</div>;
 }
+
+/**
+ *
+ * @typedef ScrollboxBaseProps
+ * @prop {boolean} [withHeader=false] - Provide layout affordances for a sticky
+ *   header in the scrollable content
+ */
+
+/**
+ * Render a scrollable container to contain content that might overflow.
+ *
+ * @param {ScrollboxBaseProps & ContainerProps} props
+ */
+export function Scrollbox({ children, classes = '', withHeader = false }) {
```

Comment:
The classes in these two components will be ignored, isn't it? `<Scrollbox className="myClass" />` or `<Scrollbox class="myClass" />`

Wouldn't be better to support and extend the native `class` and `className` props?

Reply:
I see that other components also uses `classes`.

Reply:
There is an API emerging for the components in this package. There is an issue that sort of captures the need to formalize it: https://github.com/hypothesis/frontend-shared/issues/129

I think we're moving towards all (or nearly all, still needs assessment) components accepting a few common props. The most evident one is `classes`, which is a string of any classes to append to the outermost element rendered by the component. I chose the name `classes` for some coherent reason, which I am having trouble tracking down now! The train has somewhat left the station on this (or, at least, will be a breaking change if we decide to use `className` instead). I'll see if I can't track down the origin of this naming convention; I swear I didn't make it up.

Another common prop taken by non-container components is a `ref`, and the naming convention we settled on was `<elementName>Ref`, e.g. `inputRef`, `buttonRef`.

Anyway, this definitely needs some clarity/formalization....